File: Estrutura_Sequencial/JavaScript/Ex1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) {return lines.shift();};

var raio = parseFloat(prompt());
var pi = 3.14159;

volume = (4/3) * pi * Math.pow(raio, 3);
console.log("VOLUME =", volume.toFixed(3));

File: Estrutura_Sequencial/C#/Ex1011.cs
using System;

class Program {
    static void Main(string[] args) {
        
        double raio = double.Parse(Console.ReadLine());
        double pi = 3.14159;

        double volume = (4.0 / 3.0) * pi * Math.Pow(raio, 3);
        Console.WriteLine($"VOLUME = {volume:F3}");
        
    }
}